src/styles/watches.scss

```
@@ -0,0 +1,43 @@
+.watches {
+  display: flex;
+  align-items: flex-end;
+  justify-content: center;
+  position: relative;
+  margin-top: 10px;
```

Блок і зовнішні відступи не сумісні :)
```suggestion
  padding-top: 10px;
```

---

Буду знати, виправив на локальному репозиторію)